src/Main.java

```diff
@@ -7,77 +7,119 @@
+        if (number == '0') {
+            drawLine(img, offset, true, false, true, true, true, true, true, false, false, false);
         }
-        if( w2 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 99, 82, 2);
+        if (number == '1') {
+            drawLine(img, offset, false, false, false, false, false, true, true, false, false, false);
         }
-        if( w3 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 179, 82, 2);
+        if (number == '2') {
+            drawLine(img, offset, true, true, true, false, true, true, false, false, false, false);
         }
-        if( h1 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 19, 2, 82);
+        if (number == '3') {
+            drawLine(img, offset, true, true, true, false, false, true, true, false, false, false);
         }
-        if( h2 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 99, 2, 82);
+        if (number == '4') {
+            drawLine(img, offset, false, true, false, true, false, true, true, false, false, false);
         }
-        if( h3 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(89, 19, 2, 82);
+        if (number == '5') {
+            drawLine(img, offset, true, true, true, true, false, false, true, false, false, false);
         }
-        if( h4 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(89, 99, 2, 82);
+        if (number == '6') {
+            drawLine(img, offset, true, true, true, true, true, false, true, false, false, false);
+        }
+        if (number == '7') {
+            drawLine(img, offset, true, false, false, false, false, true, true, false, false, false);
+        }
+        if (number == '8') {
+            drawLine(img, offset, true, true, true, true, true, true, true, false, false, false);
+        }
+        if (number == '9') {
+            drawLine(img, offset, true, true, true, true, false, true, true, false, false, false);
         }
     }
 
-    public static void main(String[] args) throws IOException {
-        BufferedImage img = new BufferedImage(100, 200, BufferedImage.TYPE_INT_RGB);
+    static void drawSymbol(BufferedImage img, char number, int offset) throws IOException {
+        if (number == '-') {
+            drawLine(img, offset, false, false, false, false, false, false, false, true, false, false);
+        }
+        if (number == '+') {
+            drawLine(img, offset, false, false, false, false, false, false, false, false, true, false);
+        }
+        if (number == '=') {
+            drawLine(img, offset, false, false, false, false, false, false, false, false, false, true);
+        }
+    }
+
+    static void drawLine(BufferedImage img, int offset, boolean w1, boolean w2, boolean w3, boolean h1, boolean h2, boolean h3, boolean h4, boolean e1, boolean e2, boolean e3) throws IOException {

```

Comment:
w1, w2, w3 stands for what? My idea was to call them v1, v2....., h1, h2....
v - vertical
h - horizontal

Reply:
Corrected as suggested. Instead of "w" there is "v" now, according to the English language.